test/parallel/test-whatwg-encoding.js

```diff
@@ -0,0 +1,94 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Buffer, TextDecoder, TextEncoder } = require('buffer');
```

Comment:
https://github.com/nodejs/node/blob/master/tools/eslint-rules/require-buffer.js

(added in https://github.com/nodejs/node/pull/1794)